CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
 
 ## Improvements
 
--
+- Refactored the service vdm generation and deprecated old flow.
```

Comment:
Minor: Use imperative and maybe make it less general. I guess we will need compatibility notes, but feel free to do this at the end.

Reply:
I adjusted the changelog but it is really hard for me to write aktive and this imperative is almost impossible. I spend now 15 min on a few sentences. I am simply used to either a collective `we did ABC` or `ABC has been done` in phrasing some activities.